docs/Gemfile.lock

```diff
@@ -48,28 +48,28 @@ GEM
     mercenary (0.4.0)
     pathutil (0.16.2)
       forwardable-extended (~> 2.6)
-    public_suffix (5.0.3)
-    rake (13.0.6)
+    public_suffix (5.0.4)
+    rake (13.1.0)
     rb-fsevent (0.11.2)
     rb-inotify (0.10.1)
       ffi (~> 1.0)
     rexml (3.2.6)
     rouge (3.26.0)
     safe_yaml (1.0.5)
-    sass-embedded (1.63.6)
-      google-protobuf (~> 3.23)
+    sass-embedded (1.69.7)
```

Comment:
Hi @nchammas , I hit an error when building docs for the 4.0 preview release
```
Your bundle is locked to sass-embedded (1.69.7) from rubygems repository https://rubygems.org/ or installed locally, but that version can no longer be found in that
source. That means the author of sass-embedded (1.69.7) has removed it. You'll need to update your bundle to a version other than sass-embedded (1.69.7) that hasn't been
removed in order to install.
```

I don't know the context of picking `sass-embedded` version here, which version should we use?

cc @dongjoon-hyun

Reply:
We have doc building job in PR pre-merge tests, I'm wondering how this can be broken. Or is it because the test environment is different from the release environment defined by https://github.com/apache/spark/blob/master/dev/create-release/spark-rm/Dockerfile ?

Reply:
This version was selected by Bundler. Try running `bundle update` to bump the versions in the lock file and try again. 

Would you like me to open a PR to bump the Ruby dependencies?

Reply:
oh, we upgraded bundle at https://github.com/apache/spark/commit/1e3892f939f8b71d12209d33bcb33addb0eb1879#diff-78c95b3441eb17ae275e1651c656a3614738ea96c7a13df92c95b0b987b6c497R77 but not in the docker file. let me try it.

Reply:
Ruby 2 is EOL, so we should not be [using Ruby 2.7][1] anywhere. I can also address this if you like, though it should not be a blocker. A simple refresh of the Gem lock file should be enough for now.

[1]: https://github.com/apache/spark/blob/master/dev/create-release/spark-rm/Dockerfile#L84

Reply:
it's a blocker for producing a 4.0 release...

@nchammas can you help to make necessary updates to the docker file so that it can build docs? Thanks!

Reply:
https://github.com/apache/spark/pull/46512